test/ffmpeg-qsv/transcode/transcoder.py

```diff
@@ -154,6 +169,39 @@ def check_output(self):
       "hwaccel initialisation returned error", self.output, re.MULTILINE)
     assert m is None, "Failed to use hardware decode"
 
+  # scale scaled test stream to source file width and height for generating yuv
+  def scale_to_source_size(self, scaled, codec, mode, width, height, frames, enc_file):
+    _, _, ffdecparms = self.get_requirements_data("decode", codec, mode)
+    ffdeccodec, _ = ffdecparms
+    _, _, ffencparms = self.get_requirements_data("encode", codec, mode)
+    ffenccodec, ext = ffencparms
+    # TBD - for mjpeg and mpeg2, HW scaling not working so using SW scaling
+    # Remove this once HW scaling works for jpeg and mjpeg2
```

Comment:
If it's not working, then it's a middleware bug... so why would we want to hide this bug?  Please don't hide the bug.

Reply:
removed this and added HW, SW scaling